src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,45 @@
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    align-items: center;
+    list-style: none;
+
+    &-item:not(:last-child) {
+      margin-right: 20px;
+    }
```

Comment:
You shouldn't define an element inside another element `__list-item` and `__list` although share the beginning of the name are two separate elements, using nesting here is discouraged as it may lead to misinterpretation